packages/target-truffle-v5/package.json

```diff
@@ -30,8 +30,8 @@
     "test:fix": "yarn lint:fix && yarn format:fix && yarn test && yarn typecheck"
   },
   "peerDependencies": {
-    "typechain": "^4.0.0",
```

Comment:
Can we revert this change as it's just reordering?

Reply:
Yes. I can revert.
But, `package.json` has been reordered by `yarn`. So, the diff may occur again.